Diagnostic/diagnostic.py

```diff
@@ -695,6 +695,26 @@ def start_daemon():
         hutil.error("wait daemon start time out")
 
 
+def check_suspected_memory_leak(pid):
+    memory_leak_threshold_in_KB = 2000000  # Roughly 2GB
```

Comment:
Hmm... I can definitely do that, but if you think this is really important, I think we should just do it now. And I thought of automatically calculating this based on the installed physical memory size (/proc/meminfo: "MemTotal:    xxxxxxxx kB"). Could you give me a good formula on this threshold with that information?